cmd/schema_gen.go

```diff
@@ -74,6 +88,91 @@ func generateSchemaForGraph(graph *fs.Graph) error {
 		pb.Increment()
 		return nil
 	}, config.NumberThreads(), pb)
+
+}
+
+type DocSuggestions struct {
```

Comment:
Not sure where best to define this type? Should it go in its own .go file? If so, which directory?

Reply:
@DomBlack and pointers on this? Or is this okay?

Reply:
I think this is ok for now. I wonder if we'd want a `docs` package which contained this and eventually things like the doc rendering code for creating the docs website